src/types/OIDC4VCI.types.ts

```diff
@@ -0,0 +1,55 @@
+export enum GrantTypes {
+  AUTHORIZATION_CODE = 'authorization_code',
+  PRE_AUTHORIZED = 'urn:ietf:params:oauth:grant-type:pre-authorized_code',
```

Comment:
PRE_AUTHORIZED_CODE

Reply:
This is grant type ... This suggested name will be confused with the field that contains the code.

Reply:
PRE_AUTHORIZED_CODE and AUTHORIZATION_CODE are basically referring to the same thing. 
I believe when the grant is being given the server refers to that grant as 'AUTHORIZATION_CODE '.
When we are requesting access token we refer to our grant 'PRE_AUTHORIZED'.

So the server is sending us auth code. and we are telling server to give us the access code as we are pre-authroized.

That was the sense I got from it.

But pushed the fix.

Reply:
And when were are creating the request ... we say that since our grant is 'pre-authorized' or auth-code is 'xyza' ...

Reply:
They are certainly not the same thing. Different code types for different flows

Reply:
BTW, you forgot to update the usage of the enum everywhere

Reply:
Refactor was indeed in the next commit.

Reply:
I am keeping an eye on the build in git. I hope it will be passing and all comments resolved.